source/assets/sass/components/_buttons.scss

```diff
@@ -7,10 +7,7 @@
   align-items: center;
   justify-content: center;
   margin: 0;
-  border: 0 {
```

Comment:
it's very subtle but it seems like without `border: 0` here it adds a tiny bit of space at the bottom of the buttons?

Reply:
I totally missed this width value. I was mostly staring at the nested radius. Another reason to un-nest it :)

Reply:
fixed